pkg/templates/charts/toggle/console-mce/templates/console-quickstart.yaml

```diff
@@ -0,0 +1,169 @@
+            1. Copy and paste the following YAML, then click **Create**.
+            
+                ```
+                apiVersion: cluster.open-cluster-management.io/v1beta1
+                kind: Placement
+                metadata:
+                  name: identity-placement
+                spec: {}
+                ```
+
+              An empty `spec` will select all clusters from cluster sets that are bound to the namespace. For more information on selecting clusters based on labels and other criteria, see [Using ManagedClusterSets with Placement](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html/clusters/managing-your-clusters#placement-managed).
+
+      review:
+        instructions: |-
+          Verify that the desired clusters are selected:
+
+            1. Select **local-cluster** from the cluster switcher.
+            1. Navigate to **[Home]{{ "{{" }}highlight qs-nav-home{{ "}}" }} > Search**. Search for `PlacementDecision` resources in the namespace you chose for identity management, and enter the **Label** `cluster.open-cluster-management.io/placement=identity-placement` editing it to match the name you used for your placement, if different from the example.
+            1. Click on the placement decision in the results then click on the *YAML* tab.
+
+          In the `spec.decisions` section of the YAML, do you see an entry for each cluster that you want to use with identity management?
+        failedTaskHelp: This task is incomplete. Try the task again.
+    - title: Configure an identity provider
+      description: |-
+        Configure an identity provider (like GitHub, LDAP, or OpenID) by following the instructions in the [Getting started](https://identitatem.github.io/idp-mgmt-docs/quick_start.html) documentation.
+
+        - Most types of `Secret` that you may require for your configuration can be created in the console by selecting **local-cluster** in the cluster switcher, then navigating to **[Workloads]{{ "{{" }}highlight qs-nav-workloads{{ "}}" }} > Secrets** and using the **Create** dropdown.
+        - Your `AuthRealm` resource can be created in the console by clicking the **[Import]{{ "{{" }}highlight qs-masthead-import{{ "}}" }}** button.
+        
+      review:
+        instructions: |-
+          Verify the identity provider has been configured. It can take about 5 minutes for the changes to take effect.
+          
+            1. Select a managed cluster that you chose for identity management from the cluster switcher.
+            1. If you were already logged in to the managed cluster:
+                1. Log out.
+                1. Log in to the hub cluster again.
+                1. Select the managed cluster from the cluster switcher.
+
+          Are you presented with a dialog with the text **Log in with** and a button corresponding to the name of the `AuthRealm` that you created?
```

Comment:
After you switch to the managed cluster, a dialog box with the text, **Log in with** button and a button corresponding to the name of the `AuthRealm` that you created appears.

Reply:
<img width="410" alt="image" src="https://user-images.githubusercontent.com/42188127/164555381-39b91d76-91ab-4674-ad82-ac613141f460.png">

Reply:
I recommend the following: 

Verify that the identity provider is configured. It can take about five minutes for the changes to take effect.

Reply:
Added word "that" here: https://github.com/stolostron/backplane-operator/pull/151/commits/8fc04dbf09db1a10f38c8c468225fc4b8d9f7825